src/QuadVelocityController.cpp

```diff
@@ -167,17 +213,21 @@ bool QuadVelocityController::getVelocities(geometry_msgs::Twist& return_velociti
             return_velocities.angular.z = yaw_difference / delta;
 
             // Store this as the last valid velocity
+            have_last_velocity_stamped_ = true;
             last_velocity_stamped_.twist = return_velocities;
             last_velocity_stamped_.header.stamp = time;
+
+            last_transform_stamped_ = transformStamped;
```

Comment:
These lines are duplicated in both the if and the else statement. Can they be moved out of the if statement to avoid code duplication? I don't think it improves clarity in this case to have them duplicated in the if and the else.